inputs/nuscenes/evaluate_tracking_baseline_data_test.py

```diff
@@ -2,21 +2,31 @@
 import tempfile
 
 import pytest
-
-from inputs.nuscenes.evaluate_tracking import (
+from nuscenes.eval.tracking.tooling.evaluate_tracking import (
     TrackingConfig,
     get_nuscenes_tracking_config_from_own_file,
     get_official_tracking_config,
     nuscenes_devkit_tracking_eval,
 )
-from inputs.nuscenes.nuscenes_format import TrackingEvalParams
+from nuscenes.eval.tracking.tooling.nuscenes_format import TrackingEvalParams
 
 
 @pytest.fixture
 def official_config_stripped_path():
     return "inputs/nuscenes/official_config_stripped.json"
 
 
+@pytest.fixture
+def artery_config_path():
```

Comment:
TODO: move this file to the nuscenes folder and just make it a random example file. In this unit test, it does not matter whether this is from artery or not.